### DISCUSSION TITLE
Routing for using Stripe CLI

### DISCUSSION THREAD

**Discussion by hondaman900** [spatie/laravel-stripe-webhooks#140] [Q&A]
So I'm working on my local dev copy, implementing `laravel-stripe-webhooks`. My route to my `config\stripe-webhooks.php` endpoint is:

`Route::stripeWebhooks('stripe-webhook');`

Since this work is not yet on my live site, I have not set the endpoint in the Stripe dashboard, but I do provide it in the Stripe CLI as follows:

`stripe listen --forward-to localhost:8080/DS_dev_01/config/stripe-webhooks.php`

In my call to Stripe to launch the checkout page I use the same endpoint as follows:

```
  $(document).on('click', '[id^=checkout-button-]', function(){
    var $planid = $("#"+$(this).val()).val()
    var $returnPath = window.location.origin+"/DS_dev_01/config/stripe-webhooks.php";
    alert($returnPath);
  stripe.redirectToCheckout({
    lineItems: [{
      price: $(this).val(),
      quantity: 1
    }],
    mode: 'subscription',
    successUrl: $returnPath,
    cancelUrl: 'https://www.example.com/cancel'
  });
  });
```

In my `stripe-webhooks.php` I have the standard provided code, and here's my call to the` payment_intent.succeeded` hook:

```
    'jobs' => [
        "payment_intent_succeeded" => \App\Jobs\StripeWebhooks\PaymentIntentSucceeded::class,
    ],
```

And here's the simple job helper code that is not being executed/reached:

```
<?php

namespace App\Jobs\StripeWebhooks;

use Illuminate\Bus\Queueable;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Foundation\Bus\Dispatchable;
use Illuminate\Queue\InteractsWithQueue;
use Illuminate\Queue\SerializesModels;
use Spatie\WebhookClient\Models\WebhookCall;
use Illuminate\Support\Facades\Log;
use IlluminateSupportFacadesLog;

class PaymentIntentSucceeded implements ShouldQueue
{
    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;

     /** @var \Spatie\WebhookClient\Models\WebhookCall */
     public $webhookCall;
     

    public function __construct(WebhookCall $webhookCall)
    {
        Log::info("Handle touched");

        $this->webhookCall = $webhookCall;
    }

    public function handle()
    {
        Log::info("Job Started");

        $charge = $this->webhookCall->payload['data']['object'];

        echo("Success");
        dd($charge);
        Log::info("Job Ended");
    }
}
```

Both in the Stripe CLI and Stripe Dashboard I'm seeing a `200 `success response for this event and others. The package is writing the payload to my database so I know the process from Stripe's viewpoint and the package receiving end are working, but the job is not being processed, and I'm getting no responses from either `dd()` or `log::` debug messaging inside the job code.

Since everything looks correct I'm wondering if the routing is the issue. I have my route setup as recommended for the package, but in the CLI I provide an absolute path to the endpoint, and in the session info sent to Stripe it's another concatenated path, and then there's none in the Stripe Dashboard because I'm using the Stripe CLI. Am I doing this right or is there a missing piece or conflict that allows the process to go through but never reach the job? Without the job functioning I'm dead in the water and can't process anything in my app.

Any  insight and suggestions are very welcome.

**Comment by Wyatt-NAT** [comment]
@hondaman900 have you fixed the issue? Same issue [158](https://github.com/spatie/laravel-stripe-webhooks/discussions/158)

**Comment by hashimaziz1** [comment]
Why are legitimate issues in this repository getting locked, moved and left to die without a response in the "discussions" section?
